app/src/main/java/com/example/criminalintent/presentation/dialogs/DatePickerFragment.kt

```diff
@@ -0,0 +1,52 @@
+package com.example.criminalintent.presentation.dialogs
+
+import android.app.DatePickerDialog
+import android.app.Dialog
+import android.os.Bundle
+import android.widget.DatePicker
+import androidx.fragment.app.DialogFragment
+import java.util.*
+
+class DatePickerFragment : DialogFragment() {
+
+    override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
+        val dateListener =
```

Comment:
I would move this to a `getDateListener` function in order to make this method easier to read